src/company_base.h

```diff
@@ -76,7 +76,8 @@ struct CompanyProperties {
 
 	Owner share_owners[4];           ///< Owners of the 4 shares of the company. #INVALID_OWNER if nobody has bought them yet.
 
-	Year inaugurated_year;           ///< Year of starting the company.
+	Year inaugurated_year;           ///< Year of starting the company, n .
+	GameYear inaugurated_game_year;   ///< Game year when the company was started
```

Comment:
what's the difference between inaugurated_year and inaugurated_game_year?

Reply:
Year is the actual calendar year; GameYear is the number of years since the current game began (counting up from 0).